README.md

```diff
@@ -46,21 +48,26 @@ Then follow the instructions in [INSTALL](INSTALL.md).
 
 ## Configuration
 
-For necessary server-side configurations, follow instructions in
-[INSTALL - Configuration](INSTALL.md#configuration).
+If not using the vagrant box, follow instructions in
+[INSTALL - Stand alone installation](INSTALL.md#stand-alone-installation)
+for necessary server-side configurations.
 
 ## Usage
 
-Generally you will have four tabs (or windows) open in your terminal when using this project.
-These will be used for:
+When running this project you will generally have four tabs
+(or windows) open in your terminal, which will be used for:
  1. **Git operations**.
-    Perform Git operations and general development in the repository.
+    Perform Git operations and general development in the repository,
+    such as `git checkout flapjack`.
  2. **Elasticsearch**.
```

Comment:
Add note to 2 and 3 that they are only necessary if running a local environment and not using vagrant.